### PR TITLE
Td 9

### PR DESCRIPTION
**Pull request by LeoniePhelps**
Add functionality for post a comment with user feedback

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
 export const Comments = ({ article_id }) => {
   const [comments, setComments] = useState([]);
 
   useEffect(() => {
     getCommentsByArticleId(article_id).then(({ data }) => {
+      data.sort((a, b) => {
+        let da = new Date(a.created_at);
+        let db = new Date(b.created_at);
+        return db - da;
+      });
```
Comment (douglashellowell) [inline src/components/Comments.jsx:15]: does your BE API have a query that could do this sorting for you?

sorting a lot of a data can be an expensive task that's better suited to the BE API

**Comment by douglashellowell** [comment]
amazing! works perfectly

It would be nice to see an indication of who the user is "logged in as" somewhere

**Comment by douglashellowell** [comment]
ready to merge 💪🏻

**Comment by LeoniePhelps** [comment]
Thank you!

**Pull request merged**